### DISCUSSION TITLE
How can I delete my account (hobby plan)?

### DISCUSSION THREAD

**Discussion by meloniasty** [vercel/community#7446] [Account Issue] [answered]
### Question

Hi,
How can I delete my account? I'm getting confirmation email after some minutes but when i click delete my account I have message: "The token used for issuing the deletion of your account has expired." .
I tried 4 times.

Token has too short expiration time. I have gmail email.

best regards
dm

**Comment by amyegan** [comment, marked as answer]
Hey @meloniasty. You should be able to delete your account by following this guide: 

https://vercel.com/guides/how-do-i-delete-my-vercel-account

**Reply by meloniasty** [reply]
Yes, I doing that, but email with confirmation token takes so long that when i click on buttony then is expired.

**Reply by amyegan** [reply]
Thanks for giving that a try. I'm not able to reproduce the error. I created a test account with a Gmail address and the notification arrived instantly. To simulate a delayed email, I tried waiting 15 minutes before clicking the confirmation link and it still worked for me.

Can you let me know how long you waited before you were able to click the confirmation link and share a screenshot of the error you see?

[a comment by an automated account is not shown]
